cypress/support/cypress-commands/assertion.js

```diff
@@ -101,7 +101,7 @@ Cypress.Commands.add(
                 cy.log(
                   `Actual error code ${apiErrorResponse.code} expected to be present in list of expected error codes`
                 ).then(() => {
-                  assert.include(
+                  fireLog.include(
```

Comment:
The cy log above this, that should have been internally handled by firelog, correct?